.docker/createQuery.sql

```diff
@@ -0,0 +1,141 @@
+CREATE TABLE ROOM_TYPE ( -- Typ pokoju, do typów mozna potem dodac jakies kolory/zdjecia
+    type_id INT NOT NULL,

```

Comment:
mordko chcemy robić type_id, room_id, itd?
jak dla mnie to jak podamy mu zwykłe id INT NOT NULL, to będzie wiadomo, że to chodzi o id tej encji, a np. type_id będzie sie odnosić do innej bazy

Reply:
nie uważasz, że konwencja roomId, typeId jest czytelniejsza?

Reply:
a i nie chcemy zrobić do idków sekwencji? lepiej by było, aby to się samo generowalo

Reply:
Serio... imo to może się nawet nazywać pomidoryFirmyBiedronkaId.. itak wiadomo że to jest id tej tabeli. Ten commit miał tylko dodać same entity bez ustalania walidacji/generowania... Można to dodać w jednej linijce bez większego problemu.
Co do konwencji to każda jest dobra jeśli jest czytelna - nie widzę większej różnicy między _id a id.